src/introduction/cross-diy.md

```diff
@@ -1,22 +1,10 @@
-# Cross compilation and execution
+# Manual execution
 
-## Obtaining a compiler and sysroot
+## Compiler and Sysroot Paths
 
-If you already have a compiler and sysroot (e.g. you have a docker image with pre-compiled versions), you will need to know the path to `clang` and the path to your sysroot.  You can then proceed to [**Compiler command line**](#compiler-command-line).
-
-### Building a cross build environment with cheribuild
-
-First, clone the cheribuild repo:
-```
-git clone https://github.com/CTSRD-CHERI/cheribuild.git
-```
-The [README.md](https://github.com/CTSRD-CHERI/cheribuild/blob/master/README.md) file contains considerable information, but to get started, you'll need to bootstrap an LLVM compiler and a CheriBSD build and sysroot.  The easiest path to doing this is:
-```
-cheribuild.py cheribsd-riscv64-purecap -d
-```
-This will churn away, prompting occasionally as it bootstraps assorted dependencies.  On a fast machine this will take several hours.
-<!-- XXX: Should we advocate `-f` here? -->
-Upon completion, you will find a usable Clang compiler in `~/cheri/output/sdk/bin/clang` and a sysroot in `~/cheri/output/rootfs-riscv64-purecap` (unless you have altered `cheribuild`'s default paths).
+We assume that you have followed the [prior instructions](./get.md) to obtain a
+CHERI software stack and are, therefore, in possession of file paths to both the
+`clang` compiler and a `sysroot` for your desired architecture.
```

Comment:
The text that follows this seems to contain RISC-V-isms?

Reply:
That's quite true.  I'm unlikely to spend time fixing this properly, but I can at least note the present absence of proper instructions.